### ISSUE TITLE
julia and x86 not in the list of backends

### ISSUE THREAD

**Issue by jdiaz97** [lcompilers/lpython.org-deploy#36]
They are mentioned but then dropped, this is confusing.

https://github.com/lcompilers/lpython.org-deploy/blob/2bc921d5a5dc8c94d55fa8b5340c9f2cda5373b1/content/blog/lpython_mvp.md?plain=1#L30-1#L37

**Comment by certik** [comment]
Yes, we should mention it. LFortran supports both and in LPython there might not be an option in the frontend yet, but we should add it, so that people can play with it.
